inc/environment_indicator/namespace.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Altis\Cloud\Environment_Indicator;
+
+use function Altis\get_environment_name;
+use function Altis\get_environment_type;
+
+function bootstrap() {
+	add_action( 'admin_bar_init', __NAMESPACE__ . '\\enqueue_admin_scripts' );
+	add_action( 'admin_bar_menu', __NAMESPACE__ . '\\add_admin_bar_env_info' );
+}
+
+/**
+ * Add menu item to admin bar to display environment info:
+ * - environment type
+ * - environment name
+ * - URL to Altis dashboard for the environment
+ */
+function add_admin_bar_env_info() {
```

Comment:
The `admin_bar_menu` action passes the admin bar class through so you don't need the `global $wp_admin_bar;` in there eg:
 ```suggestion
function add_admin_bar_env_info( WP_Admin_Bar $wp_admin_bar ) {
```

You'll need to `use` the `WP_Admin_Bar` class after the namespace declaration.